index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TechHub-Careers</title>
+    <!-- Css  -->
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="nav-container">
+            <div id="nav-branding">
+                <h1>Tech<span>Hub</span>-Careers</h1>
+            </div>
+            <nav>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="#">Careers</a></li>
+                    <li><a href="#">Wait-List</a></li>
```

Comment:
According to the question, `waitlist` should be one word
```suggestion
                    <li><a href="#">Waitlist</a></li>
```